colossalai/fx/_compatibility.py

```diff
@@ -0,0 +1,37 @@
+import torch
+
+from . import META_COMPATIBILITY
+
+
+def compatibility(is_backward_compatible: bool = False):
+    """A decorator to make a function compatible with different versions of PyTorch.
+
+    Args:
+        is_backward_compatible (bool, optional): Whether the function is backward compatible. Defaults to False.
+
+    Returns:
+        Callable: The decorated function
+    """
+
+    def decorator(func):
+        if META_COMPATIBILITY:
+            return func
+        else:
+            if is_backward_compatible:
+                return func
+            else:
+                return lambda *args, **kwargs: print(
+                    f'{func} seems to be incompatible with PyTorch {torch.__version__}.')
+
+    return decorator
```

Comment:
Hopefully this will do the checking and replace the function that are not backward_compatible into a print statement